src/styles/blocks/body.scss

```diff
@@ -0,0 +1,12 @@
+html {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
it would be better to name this file 'reset' or 'normalize'

Reply:
✅